components/Banner/src/MDCBannerView.h

```diff
@@ -16,6 +16,14 @@
 
 #import "MaterialButtons.h"
 
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutMode) {
+  MDCBannerViewLayoutModeAutomatic,   // Layout is set automatically based on how elements are
+                                      // configured on banner view
```

Comment:
Will the automatic mode ultimately choose one of the modes below behind the scenes? Please document the behavior here.

Reply:
Yes, it does. I will add the document. The bannerViewLayoutMode, which is public API, always keeps the user preference.

Reply:
Addressed in b5d725fdd272ec093f273a732c8948206c06f345.